### PR TITLE
fix(scheduler): update otel-collector-config after jaeger endpoint does not supported

### PR DESCRIPTION
**Pull request by ducminhle**
**What this PR does / why we need it**:
With existing config: otel-collector-config.yaml, I got an error when try to run the `make deploy-local`, `otel-collector` container crashed with logs:

```
Error: failed to get config: cannot unmarshal the configuration: 1 error(s) decoding:

* error decoding 'exporters': unknown type: "jaeger" for id: "jaeger" (valid values: [file prometheusremotewrite skywalking splunk_hec cassandra dataset influxdb kafka awss3 otlp azuremonitor googlecloudpubsub loadbalancing zipkin logging datadog elasticsearch sapm sentry sumologic tanzuobservability tencentcloud_logservice debug carbon clickhouse mezmo prometheus azuredataexplorer awsxray f5cloud googlecloud pulsar syslog alibabacloud_logservice coralogix logzio loki opensearch awskinesis awscloudwatchlogs awsemf dynatrace googlemanagedprometheus honeycombmarker instana logicmonitor otlphttp signalfx opencensus])
2023/12/11 06:42:49 collector server run finished with error: failed to get config: cannot unmarshal the configuration: 1 error(s) decoding:

* error decoding 'exporters': unknown type: "jaeger" for id: "jaeger" (valid values: [file prometheusremotewrite skywalking splunk_hec cassandra dataset influxdb kafka awss3 otlp azuremonitor googlecloudpubsub loadbalancing zipkin logging datadog elasticsearch sapm sentry sumologic tanzuobservability tencentcloud_logservice debug carbon clickhouse mezmo prometheus azuredataexplorer awsxray f5cloud googlecloud pulsar syslog alibabacloud_logservice coralogix logzio loki opensearch awskinesis awscloudwatchlogs awsemf dynatrace googlemanagedprometheus honeycombmarker instana logicmonitor otlphttp signalfx opencensus])
```

I found the problem: `opentelemetry-collector-contrib` has removed jaeger and jaegerthrifthttp exporters from this commit: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26546. We need to update the endpoint to fix this issue (https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15291#issuecomment-1826734333)

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @ducminhle. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) or [todo](https://github.com/orgs/todo/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/todo/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

File: scheduler/all-base.yaml [hunk 1]
```diff
@@ -103,6 +103,7 @@ services:
       - "16686:16686"
       - "14268"
       - "14250"
```
Comment (sakoush) [inline scheduler/all-base.yaml:105]: do we still need port `14250`? It feels that it is being replaced by `4317`

**Review by sakoush — CHANGES_REQUESTED** [review]
@ducminhle many thanks for your contributions, this looks good in general. I left one comment on whether we still need to expose port `14250`?

This PR covers docker compose installations, however the same issue you are fixing exists potentially with installations in k8s, e.g. check: `tracing/k8s/otel-collector.yaml`. If you are willing to fix it as well in k8s that would be great, but happy to leave it to a separate contribution.

Please also outline the tests that you have done and any screenshots you might have. e.g. to make sure that traces appear in `jeager` ui after this change.

**Comment by ducminhle** [comment]
image
image
image
image
image

**Comment by seldondev** [comment]
@cin-otis: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

<details>

In response to [this](https://github.com/SeldonIO/seldon-core/pull/5170#issuecomment-1884273112):

>/ok-to-test


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by seldondev** [comment]
@ducminhle: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

<details>

In response to [this](https://github.com/SeldonIO/seldon-core/pull/5170#issuecomment-1884273583):

>/ok-to-test


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by lc525** [comment]
I believe this is now solved via #5291, which includes changes required for things to work on k8s, as well as in local deployments. Many thanks for your contribution, and for bringing the issue to our attention.

**Pull request closed without merging**